### ISSUE TITLE
[raw query][postgresql]invalid placeholder index 

### ISSUE THREAD

**Issue by KumanoTanaka** [Brendonovich/prisma-client-rust#335]
I'm using postgresql and prisma-client-rust.

```toml
prisma-client-rust = { git = "https://github.com/Brendonovich/prisma-client-rust", tag = "0.6.8", features = ["postgresql", "migrations"], default_features = false }
```

I faced the sql error when I tried to use the `_query_raw` feature.

I think the code below should look like ...

https://github.com/Brendonovich/prisma-client-rust/blob/0.6.8/src/raw.rs#L45

```rust
        for i in 0..values.len() {
            let variable_indicator = match database {
                "postgresql" => format!("${i + 1}"), // place holder should start with 1 I guess...
```

**Comment by Brendonovich** [comment]
Should be fixed in `rev = "b982a03f5393c116df52b1d2c45daca7d6d859d4"` (though a lot of other changes are also at that commit haha)

**Comment by KumanoTanaka** [comment]
thank you so much
waiting for the next release
